code/game/gamemodes/autotraitor/autotraitor.dm

```diff
@@ -56,32 +49,27 @@
 			traitor.special_role = SPECIAL_ROLE_TRAITOR
 			traitor.restricted_roles = restricted_jobs
 
-//	if(!length(traitors))
-//		return 0
-	return 1
-
-
-
+	return TRUE
 
 /datum/game_mode/traitor/autotraitor/post_setup()
 	..()
-	traitorcheckloop()
+	traitor_check_loop()
 
-/datum/game_mode/traitor/autotraitor/proc/traitorcheckloop()
+/datum/game_mode/traitor/autotraitor/proc/traitor_check_loop()
 	spawn(9000)
```

Comment:
Think it's possible you could turn this into a timer rather than a spawn(9000)

Reply:
Done!